micromasters/serializers.py

```diff
@@ -0,0 +1,78 @@
+"""
+Serializers for Django contrib models
+"""
+import logging
+from rest_framework import serializers
+from django.contrib.auth.models import User, AnonymousUser
+from django.core.exceptions import ObjectDoesNotExist
+from rolepermissions.verifications import has_role
+from roles.models import Instructor, Staff
+from backends.edxorg import EdxOrgOAuth2
+
+log = logging.getLogger(__name__)
+
+
+class UserSerializer(serializers.ModelSerializer):
+    """Serializer for users."""
+    first_name = serializers.SerializerMethodField()
+    last_name = serializers.SerializerMethodField()
+    preferred_name = serializers.SerializerMethodField()
+    is_staff = serializers.SerializerMethodField()
+    social_username = serializers.SerializerMethodField()
+
+    class Meta:
+        model = User
+        fields = (
+            "username", "email", "social_username",
```

Comment:
Can we leave out `username` here and rename `social_username` to `username`? The only username the JS client should know about is the social one which is the one from edX. Often `User.username` will be the same but it's not guaranteed, especially if there was a backend error when logging in